ParseTwitterUtils/Tests/Unit/OAuth1FlowDialogTests.m

```diff
@@ -79,32 +73,6 @@ - (void)testKeyboard {
     [flowDialog dismissAnimated:NO];
 }
 
-- (void)testRotation {
```

Comment:
Maybe you wanna leave the test but disable it? So we now there is a case that we do not cover and may cover with UITest in the future.

Reply:
Sure, I'll make it skip instead.